Bio/PDB/MMCIF2Dict.py

```diff
@@ -47,6 +49,8 @@ def __init__(self, filename):
                     self[key] = token
                     key = None
 
+    # Private methods
```

Comment:
Methods with a single leading underscore are by Python convention private, so the comment is redundant (but harmless)